builtin/logical/pki/acme_state.go

```diff
@@ -17,12 +17,13 @@ import (
 	"time"
 
 	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/helper/nonce"
 	"github.com/hashicorp/vault/sdk/logical"
 )
 
 const (
 	// How long nonces are considered valid.
-	nonceExpiry = 15 * time.Minute
+	nonceExpiry = 15 * time.Second
```

Comment:
Should this not be restored to Minute?

Reply:
Maybe we could default to 90 seconds? I think 15 minutes is too long tbh -- an unauthenticated user can submit nonce redemptions via the new account creation API, which would result in increased memory usage.

One option could be to validate requests have valid EAB (when that is required) _before_ redeeming the nonce, but that would require some restructuring.

15 minutes seems too long, given that clients will refetch new nonces if they have expired IMO. 

Thoughts?

Reply:
I'm okay with 90 seconds, 15 seconds just seemed a little aggressive.